src/styles/cylc/_menu.scss

```diff
@@ -38,4 +38,7 @@
       line-height: 1.35;
     }
   }
+  .pad-button {
+    padding: 30px;
+  }
```

Comment:
There is no element with the class `pad-button` so this does nothing

Reply:
I did not want to commit this file - this was something I played with and discarded.